control/control.rnc

```diff
@@ -494,6 +494,20 @@ network = element network {
 
 # network
 
+# system_roles
+
+system_roles = element system_roles {
+    LIST,
+    system_role+
+}
+
+system_role = element system_role {
+    element id { text } &
+    partitioning? &
+    software?
```

Comment:
so additional kernel parameters is not included? I think it is another good and easily to support feature as bootloader already support it and it allows to e.g. use different IO scheduler for given role.